<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterUnlockCode</class>
 <widget class="EnterUnlockCode" name="EnterUnlockCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bgWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#bgWidget { background: rgba(0,0,0,220); }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutMain" stretch="1">
      <property name="leftMargin">
       <number>60</number>
      </property>
      <property name="topMargin">
       <number>60</number>
      </property>
      <property name="rightMargin">
       <number>60</number>
      </property>
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="contentWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>486</width>
          <height>350</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>648</width>
          <height>480</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#contentWidget { background: rgba(255,255,255,240); border-radius: 6px; }

QLabel { color: rgb(40,40,40);  }</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutSub" stretch="0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutUnlockCode">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayoutIcon">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../merit.qrc">:/icons/merit</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="margin">
                <number>15</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="infoText">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>6</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#064356;&quot;&gt;Merit&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#5b5b5b;&quot;&gt;Is Invite Only&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="textFormat">
                <enum>Qt::RichText</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayoutAlias">
             <property name="topMargin">
              <number>0</number>
             </property>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="unlockCodeDescription">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The person who invited you is an &lt;span style=&quot; font-weight:600;&quot;&gt;Ambassador&lt;/span&gt; of &lt;span style=&quot; font-weight:600;&quot;&gt;Merit.&lt;/span&gt;  They will be notified when you join.&lt;/p&gt;&lt;p&gt;They will also be entered into a lottery and can win &lt;span style=&quot; font-weight:600;&quot;&gt;Merit&lt;/span&gt; as you use the system. You can also become and &lt;span style=&quot; font-weight:600;&quot;&gt;Ambassador&lt;/span&gt; by inviting other and you too will have a chance to win &lt;span style=&quot; font-weight:600;&quot;&gt;Merit&lt;/span&gt; each block!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Enter the Merit Address of the person who invited you.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="unlockCodeTextInput">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The person who invited you is an &lt;span style=&quot; font-weight:600;&quot;&gt;Ambassador&lt;/span&gt; of &lt;span style=&quot; font-weight:600;&quot;&gt;Merit.&lt;/span&gt;  They will be notified when you join.&lt;/p&gt;&lt;p&gt;They will also be entered into a lottery and can win &lt;span style=&quot; font-weight:600;&quot;&gt;Merit&lt;/span&gt; as you use the system. You can also become and &lt;span style=&quot; font-weight:600;&quot;&gt;Ambassador&lt;/span&gt; by inviting other and you too will have a chance to win &lt;span style=&quot; font-weight:600;&quot;&gt;Merit&lt;/span&gt; each block!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="accessibleDescription">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>34</number>
             </property>
             <property name="placeholderText">
              <string>Enter a valid Merit address or Alias here (required)</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="aliasDescription">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Other users of &lt;span style=&quot; font-weight:600;&quot;&gt;Merit&lt;/span&gt; can use your &lt;span style=&quot; font-weight:600;&quot;&gt;Public Alias&lt;/span&gt; instead of an address to send your funds. This alias is &lt;span style=&quot; font-style:italic;&quot;&gt;optional&lt;/span&gt;. An alias cannot be changed for an address once it is chosen.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;The alias must be &lt;span style=&quot; font-weight:600;&quot;&gt;globally unique&lt;/span&gt;, nobody else can use your alias once it is commited to the Merit Blockchain.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;For more information, visit: &lt;a href=&quot;http://merit.me/aliases.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;merit.me/aliases.html&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Choose an optional Public Alias.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="aliasTextInput">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Other users of &lt;span style=&quot; font-weight:600;&quot;&gt;Merit&lt;/span&gt; can use your &lt;span style=&quot; font-weight:600;&quot;&gt;Public Alias&lt;/span&gt; instead of an address to send your funds. This alias is &lt;span style=&quot; font-style:italic;&quot;&gt;optional&lt;/span&gt;. An alias cannot be changed for an address once it is chosen.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;The alias must be &lt;span style=&quot; font-weight:600;&quot;&gt;globally unique&lt;/span&gt;, nobody else can use your alias once it is commited to the Merit Blockchain.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;For more information, visit: &lt;a href=&quot;http://merit.me/aliases.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;merit.me/aliases.html&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="placeholderText">
                <string>Enter an Alias here (optional)</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="submitButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Join the Merit Network</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnterUnlockCode</class>
   <extends>QWidget</extends>
   <header>modaloverlay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../merit.qrc"/>
 </resources>
 <connections/>
</ui>
